docs/prod-host-setup.md

```diff
@@ -24,8 +24,8 @@ To set up the jailer correctly, you'll need to:
   their individually owned resources in the unlikely case where any one of the
   jails is broken out of.
 
-- Use Jailer's ``--seccomp-level 2`` flag to enable seccomp filter. The Jailer
-  will apply a restrictive filter on what ``syscall`` and associated call
+- Use Jailer's ``--seccomp-level 2`` extra argument to enable seccomp filter. The
```

Comment:
I don't think we need this phrase here any more because seccomp level is by default 2 so the user doesn't have to do anything.

Reply:
I agree :+1: